model_zoo/models/autoencoder_cifar10/model_autoencoder_cifar10.prototext

```diff
@@ -47,18 +43,30 @@ model {
   #######
   layer {
     name: "data"
-    children: "encode1 reconstruction"
+    children: "image label"
```

Comment:
Calling the second child of data layer "label" seems like a misnomer to me. It is not "label", rather a replica of input data, can we find a clearer/meaningful name?